src/components/OperationWizard.vue

```diff
@@ -125,26 +157,45 @@ function goForward() {
       </v-row>
     </v-card-text>
     <v-slide-y-reverse-transition hide-on-leave>
-      <v-card-text v-show="page == 'configure'">
-        <v-row
+      <v-card-text
+        v-show="page == 'configure'"
+        class="wizard-card"
+      >
+        <div
           v-for="(inputDescription, inputKey) in selectedOperationInputs"
           :key="inputKey"
         >
-          <p
-            v-if="inputDescription.type == 'file'"
-            class="mt-4 mb-4"
-          >
-            {{ inputKey }}: Insert a filepicker widget for {{ inputDescription.name }}
-          </p>
           <v-text-field
-            v-if="inputDescription.type != 'file'"
+            v-if="inputDescription.type !== 'file'"
             v-model="selectedOperationInput[inputKey]"
             :label="inputDescription.name"
             :type="inputDescription.type"
+            class="operation-input"
           />
-        </v-row>
+          <div
+            v-else-if="inputDescription.type == 'file'"
+            class="images-container"
+          >
+            <v-col
```

Comment:
I didn't know v-col worked without being inside a v-row

Reply:
the more you know